<launch>
  <node type="usb_stereo_node" pkg="lfd_vision" name="usb_stereo" output="screen">
    <param name="pkg_dir" value="$(find lfd_vision)/" />
    <param name="cam_config" value="config/usbcam2s1.yaml" />
    <param name="stereo_config" value="config/usbcam2s1.yaml" />
    <param name="stereof_config" value="config/usbcam2s1.yaml" />

    <param name="num_detectors" value="2" />
    <param name="block_area_min" value="1.0" />
    <param name="color_files_base1" value="$(find lfd_vision)/x" />
    <param name="color_files_base2" value="$(find lfd_vision)/x2" />
    <param name="img_win" value="1111111" type="str" />
    <param name="disp_fps" value="30.0" />
    <param name="disp_scale" value="0.6" />

    <param name="ff_ofl_win" value="3" />
    <param name="ff_ofl_spd_min" value="3.0" />
    <param name="ff_er_dl" value="1" />
    <param name="ff_amt_min" value="3.0" />
    <param name="ff_amt_max" value="3000.0" />
    <param name="ff_spd_min" value="1.0" />
    <param name="ff_spd_max" value="-1.0" />
    <param name="ff_mask_flen" value="0" />
    <param name="send_raw_flow" value="0" />
  </node>
</launch>
